geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/parallel/ParallelWANStatsDUnitTest.java

```diff
@@ -102,15 +108,18 @@ public void testQueueSizeInSecondaryBucketRegionQueuesWithMemberRestart() throws
         (ArrayList<Integer>) vm7.invoke(() -> WANTestBase.getSenderStats("ln", NUM_PUTS));
 
     assertEquals(NUM_PUTS, v4List.get(0) + v5List.get(0) + v6List.get(0) + v7List.get(0)); // queue
-                                                                                           // size
-    assertEquals(NUM_PUTS * 2, v4List.get(1) + v5List.get(1) + v6List.get(1) + v7List.get(1)); // eventsReceived
-    assertEquals(NUM_PUTS * 2, v4List.get(2) + v5List.get(2) + v6List.get(2) + v7List.get(2)); // events
-                                                                                               // queued
+    // size
+    assertEquals(NUM_PUTS * 2,
+        v4List.get(1) + v5List.get(1) + v6List.get(1) + v7List.get(1)); // eventsReceived
+    assertEquals(NUM_PUTS * 2,
+        v4List.get(2) + v5List.get(2) + v6List.get(2) + v7List.get(2)); // events
+    // queued
     assertEquals(0, v4List.get(3) + v5List.get(3) + v6List.get(3) + v7List.get(3)); // events
-                                                                                    // distributed
-    assertEquals(NUM_PUTS, v4List.get(10) + v5List.get(10) + v6List.get(10) + v7List.get(10)); // secondary
-                                                                                               // queue
-                                                                                               // size
+    // distributed
+    assertEquals(NUM_PUTS,
+        v4List.get(10) + v5List.get(10) + v6List.get(10) + v7List.get(10)); // secondary
+    // queue
+    // size
```

Comment:
Can these comments be cleaned up a bit?

Reply:
I have removed them. I do not think they provide any useful information.